docs/02_cleos/02_how-to-guides/how-to-stake-resource.md

```diff
@@ -1,28 +1,110 @@
-## Goal
+## Overview
 
-Stake resource for your account
+This how-to guide provides instructions on how to stake resources, NET and/or CPU, for your account using the `cleos` CLI tool.
 
 ## Before you begin
 
-* Install the currently supported version of `cleos`
+* Install the currently supported version of `cleos`.
 
-* Ensure the reference system contracts from `eosio.contracts` repository is deployed and used to manage system resources
+* Ensure the [reference system contracts](https://developers.eos.io/manuals/eosio.contracts/v1.9/build-and-deploy) are deployed and used to manage system resources.
 
 * Understand the following:
-  * What is an account
-  * What is network bandwidth
-  * What is CPU bandwidth
+  * What is an [account](https://developers.eos.io/welcome/latest/glossary/index/#account).
+  * What is [NET bandwidth](https://developers.eos.io/manuals/eosio.contracts/latest/key-concepts/net).
+  * What is [CPU bandwidth](https://developers.eos.io/manuals/eosio.contracts/latest/key-concepts/cpu).
 
-## Steps
+## Command Reference
 
-Stake 0.01 SYS network bandwidth for `alice`
+See the following reference guides for command line usage and related options for the `cleos` command:
+
+* The [`delegatebw` cleos sub-command](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/system/system-delegatebw).
+
+## Procedure
+
+The following steps show:
+
+1. [How to stake NET bandwidth.](#1-stake-net-bandwidth)
+2. [How to stake CPU bandwidth.](#2-stake-cpu-bandwidth)
+3. [How to stake NET and CPU bandwidth.](#3-stake-net-and-cpu-bandwidth)
```

Comment:
Since these steps can be performed in any order, we shouldn't use numbered bullets.